controlplane/kubeadm/internal/workload_cluster_conditions.go

```diff
@@ -326,27 +350,57 @@ func compareMachinesAndMembers(controlPlane *ControlPlane, members []*etcd.Membe
 				Reason:  controlplanev1.KubeadmControlPlaneMachineEtcdMemberNotHealthyV1Beta2Reason,
 				Message: fmt.Sprintf("Etcd doesn't have an etcd member for Node %s", machine.Status.NodeRef.Name),
 			})
+
+			// Instead, surface there is a machine without etcd member on kcp's' Available condition
+			// only if the machine is not deleting and the node exists by more than two minutes
+			// (this prevents the condition to flick during scale up operations).
+			// Note: Two minutes is an arbitrary interval where we expect the system to detect the new etcd member on the machine.
```

Comment:
```suggestion
			// Note: Two minutes is the time after which we expect the system to detect the new etcd member on the machine.
```
arbitrary would basically mean that we chose the time entirely randomly